token/cli/src/main.rs

```diff
@@ -1916,6 +1919,104 @@ async fn command_sync_native(
     })
 }
 
+// Both enable_required_transfer_mesos and disable_required_transfer_mesos
+// Switches with enable_memos bool
+async fn command_required_transfer_memos(
+    token_account_address: Pubkey,
+    owner: Pubkey,
+    bulk_signers: BulkSigners,
+    enable_memos: bool,
+    config: &Config<'_>,
+) -> CommandResult {
+    if config.sign_only {
+        panic!("Config can not be sign only for enabling/disabling required transfer memos.");
+    }
+    let account_fetch = config
+        .rpc_client
+        .get_account(&token_account_address)
+        .await
+        .map_err(|err| {
+            format!(
+                "Token account {} does not exist: {}",
+                token_account_address, err
+            )
+        })?;
+    let program_id = account_fetch.owner;
+    config.check_owner(&token_account_address, &program_id)?;
+    let mut instructions: Vec<Instruction> = Vec::new();
+    // Reallocation (if needed)
+    let current_account_len = account_fetch.data.len();
+    let state_with_extension =
+        StateWithExtensionsOwned::<Account>::unpack(account_fetch.data.clone())?;
```

Comment:
nit: since you're not using `account_fetch` later, there's no need to clone the data, just move the value in

Reply:
resolved with 484279b